iam.tf

```diff
@@ -136,8 +145,18 @@ resource "google_secret_manager_secret_iam_binding" "gitlab_initial_root_pwd" {
 
 
 resource "google_secret_manager_secret_iam_binding" "gitlab_backup_key" {
-  project    = google_secret_manager_secret.gitlab_backup_key.project
-  secret_id  = google_secret_manager_secret.gitlab_backup_key.secret_id
+  project    = var.project_id
```

Comment:
Why not to give one project wide permission as "secretAccessor"?

Reply:
Because I don't want the service account to other secrets in the project.

Reply:
From what I understand your concern is that there will be multiple deployments in the same project.
Please next time elaborate when you reply.

My proposal is to give permissions with policy, that way you can give permissions to resources based on their tags, name, etc.

Please add a task to handle it.

Reply:
Added